org-journal.el

```diff
@@ -403,12 +403,24 @@ This runs once per date, before `org-journal-after-entry-create-hook'.")
 
 (defvar org-journal--search-buffer "*Org-journal search*")
 
-
+(defvar org-journal-dont-mark-entries nil)
+
 ;;;###autoload
 (add-hook 'calendar-today-visible-hook 'org-journal-mark-entries)
 ;;;###autoload
 (add-hook 'calendar-today-invisible-hook 'org-journal-mark-entries)
 
+(defun org-journal--org-read-date (orig-fun &rest args)
```

Comment:
This seems to permanently advice `org-read-date` to always see `org-journal-dont-mark-entries` as `t`. Shouldn't this be contingent on `org-journal-dont-mark-entries` actually being set. And in that case, why do we need the `advice` anyway?

I guess I'm saying I don't quite get what this is doing. Could you explain it in a bit more detail?